integrations/purchase/bridge-api-categories.js

```diff
@@ -0,0 +1,203 @@
+
+  162: purchaseActivity(PURCHASE_CATEGORY_RECREATIONAL_SERVICES), // - Shopping
+  441888: purchaseActivity(), // Licences
+  319: purchaseActivity(PURCHASE_CATEGORY_RECREATIONAL_SERVICES), // Movies
+  318: purchaseActivity(PURCHASE_CATEGORY_RECREATIONAL_DURABLES), // Music
+  272: purchaseActivity(PURCHASE_CATEGORY_STORE_CLOTHING), // Clothing & Shoes
+  262: purchaseActivity(PURCHASE_CATEGORY_RECREATIONAL_DURABLES), // Sporting goods
+  243: purchaseActivity(PURCHASE_CATEGORY_STORE_BOOKS), // Books
+  186: purchaseActivity(PURCHASE_CATEGORY_RECREATIONAL_DURABLES), // Shopping - Others
+  184: purchaseActivity(PURCHASE_CATEGORY_STORE_HARDWARE), // Hardware
+  183: purchaseActivity(PURCHASE_CATEGORY_RECREATIONAL_DURABLES), // Gifts
+
+  171: purchaseActivity(PURCHASE_CATEGORY_INFORMATION_COMMUNICATION), // - Bills & Utilities
+  280: purchaseActivity(PURCHASE_CATEGORY_INFORMATION_COMMUNICATION), // Subscription - Others
+  277: purchaseActivity(PURCHASE_CATEGORY_INFORMATION_COMMUNICATION), // Mobile phone
+  258: purchaseActivity(PURCHASE_CATEGORY_INFORMATION_COMMUNICATION), // Home phone
+  219: purchaseActivity(PURCHASE_CATEGORY_INFORMATION_COMMUNICATION), // Cable TV
+  180: purchaseActivity(PURCHASE_CATEGORY_INFORMATION_COMMUNICATION), // Internet
+
+  315: purchaseActivity(PURCHASE_CATEGORY_STORE_PERSONAL_CARE), // - Personal care
+  321: purchaseActivity(PURCHASE_CATEGORY_STORE_PERSONAL_CARE), // Beauty care
+  317: purchaseActivity(PURCHASE_CATEGORY_STORE_PERSONAL_CARE), // Personal care - Others
+  316: purchaseActivity(PURCHASE_CATEGORY_STORE_PERSONAL_CARE), // Spa & Massage
+  248: purchaseActivity(PURCHASE_CATEGORY_STORE_PERSONAL_CARE), // Cosmetics
+  235: purchaseActivity(PURCHASE_CATEGORY_STORE_PERSONAL_CARE), // Hairdresser
+
+  161: purchaseActivity(PURCHASE_CATEGORY_HOUSING), // - Home
+  328: purchaseActivity(PURCHASE_CATEGORY_MISC_DWELLING), // Misc. utilities
+  323: purchaseActivity(PURCHASE_CATEGORY_STORE_GARDEN_AND_PET), // Lawn & Garden
+  293: purchaseActivity(PURCHASE_CATEGORY_MISC_DWELLING), // Water
+  246: purchaseActivity(), // Home insurance
+  222: purchaseActivity(PURCHASE_CATEGORY_HOUSEHOLD_MAINTENANCE), // Maintenance
+  218: purchaseActivity(PURCHASE_CATEGORY_GAS), // Gas
+  217: purchaseActivity(PURCHASE_CATEGORY_ELECTRICITY), // Electricity
+
+  166: purchaseActivity(), // - Business services
+  221: purchaseActivity(PURCHASE_CATEGORY_STORE_FURNISHING), // Office improvement
+  220: purchaseActivity(PURCHASE_CATEGORY_NON_DURABLE_HOUSEHOLD_GOODS), // Offices - Others
+  216: purchaseActivity(PURCHASE_CATEGORY_RENTAL), // Office Rent
+  441900: purchaseActivity(), // Marketing
+  441899: purchaseActivity(), // Legal Fees
+  441898: purchaseActivity(), // Training taxes
+  441897: purchaseActivity(), // Disability Insurance
+  441896: purchaseActivity(), // Outsourcing
+  441895: purchaseActivity(), // Consulting
```

Comment:
Services (Marketing, Legal fees, outsourcing, consulting, accounting, hiring fees, advertising, etc) should be categorised as "Other services n.e.c."

Reply:
Hhm I would tend to agree with insurance, even though the emission factor seem off:

If you pay 5000€ to insure your house on a given year in FR, the carbon footprint will be: 5000 * 0.12 = 600kg, that just seems like a lot

Regarding Other Services N.E.C, 1st, it's not in the footprints.yml 😅 and 2nd it's either in the overall services for the dwelling or in general other services, along with ... prostitution and religious services 😅😅😅 so that's why I was careful not to use it

<img width="599" alt="Screenshot 2020-09-23 at 09 58 36" src="https://user-images.githubusercontent.com/32778266/93983821-5b264800-fd83-11ea-9242-30ec09e1105d.png">
<img width="604" alt="Screenshot 2020-09-23 at 09 59 00" src="https://user-images.githubusercontent.com/32778266/93983879-69746400-fd83-11ea-8681-2d61bf4577f3.png">